### ISSUE TITLE
open in elementary terminal launches in home directory, not current directory

### ISSUE THREAD

**Issue by ahmedmagdyy** [linuxmint/nemo#2132]
```
 * Nemo version (nemo --version) 3.6.5
 * Is issue with desktop or windowed nemo? 
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Elementary OS 5.0
 * Graphics hardware *and* driver used
 * 32 or 64 bit 64
 ```

**Issue**
after installing Nemo and set it as the default file manager, and modified the default terminal to launch pantheon terminal when i use "open in terminal" it starts the terminal but in home directory not in the current directory.

**Expected behaviour**
expected terminal starts in the current directory, not in home directory

**Comment by JosephMcc** [comment]
This functionality works fine in Mint Cinnamon and other DE's I've tried like Xfce. We've had a few other reports of Nemo not working correctly in Elementary in the past. I think Elementary is built to work best with the applications that are built for it. As far as I know there is no dev who will be running Nemo there so I'm not sure much will happen with this.

**Comment by xenopeek** [comment]
For making Nemo use a different terminal than gnome-terminal this should be configured through gsettings when not on Cinnamon. At least I know from previous issues that running `gsettings get org.cinnamon.desktop.default-applications.terminal exec 'konsole'` on KDE or running `gsettings get org.cinnamon.desktop.default-applications.terminal exec 'xfce4-terminal'` on Xfce is enough to make Nemo aware of native terminals on those DEs.

I tried to do such on elementary 5.1.4 with:
```
gsettings get org.cinnamon.desktop.default-applications.terminal exec 'io.elementary.terminal'
gsettings get org.cinnamon.desktop.default-applications.terminal exec-arg '-n'
```
I added the latter to make it open a new window. However this doesn't work:

- If there is no elementary terminal open, using the Open in Terminal function from Nemo does open a terminal and indeed on home directory not the current directory in Nemo;
- If a elementary terminal was already open then instead using the Open in Terminal function from Nemo sets focus to the already open terminal.

I note that elementary 5.1.4 is using an ancient Nemo 3.6.5 version. Whether this is an issue in Nemo 3.6.5, in elementary 5.1.4, in elementary terminal, or some combination—I don't know. **Workaround:** I suggest for now, till elementary 6 arrives which should have Nemo 4.4.2 at least, to run `gsettings reset-recursively org.cinnamon.desktop.default-applications.terminal` to reset the settings and install gnome-terminal. Nemo will use that as default and that works correctly in elementary 5.1.4.
